### PR TITLE
display clan id in api request

### PR DESCRIPTION
**Pull request by tsuuki**
idk y this wasn't done before

### CODE REVIEW

**Comment by cmyui** [comment]
good catch, but it seems a bit weird to only include clan id. i think if we're going to include clan info, we should include the full spread of what's useful for frontend.

probably should be done by fetching the clan from the player, and calling format_clan_basic() on it.

if you're up for it, try it out; if not, I can add to this PR later and we can merge it :)

**Comment by tsuuki** [comment]
i'm a tad busy but i'll try it out if you don't in the next coming days

File: app/api/domains/api.py [hunk 1]
```diff
@@ -145,14 +145,14 @@ async def api_get_player_info(
     if username:
         user_info = await app.state.services.database.fetch_one(
             "SELECT id, name, safe_name, "
-            "priv, country, silence_end "
+            "priv, clan_id, country, silence_end "
```
Comment (alowave223) [inline app/api/domains/api.py:148]: for what is even silence_end put here?

Comment (cmyui) [reply app/api/domains/api.py:148]: on osu/ripple frontend u can see how long someone is silenced for, so i thought i'd include it, public enough data

**Comment by alowave223** [comment]
i think, it would be much better if we use `Player` object instead of raw sql data, we have `from_cache_or_sql` function and in my opinon it looks better and scalable in future without any need at editing sql and parsing raw sql data, also change a `format_player_basic` little, and add new properties `safe_name` and `silence_end` since we need to get user_info either way, also we already have clan parsed with `format_clan_basic` in it.

**Comment by cmyui** [comment]
going to merge this and open an issue to add the clan object

**Pull request merged**